jsonexample/lib/built_value/built_complex_object.g.dart

```diff
@@ -19,7 +19,7 @@ part of 'built_complex_object.dart';
 // ignore_for_file: unnecessary_new
 
 Serializer<BuiltComplexObject> _$builtComplexObjectSerializer =
-    new _$BuiltComplexObjectSerializer();
+    _$BuiltComplexObjectSerializer();
```

Comment:
Did you edit these files by hand (the ones ending with `.g.dart`), or rerun the built_value source generation routine?

Reply:
I've done these edits manually. read and understood how json_serializable works.  will revert changes made on the generated files. :) 👍